src/Symfony/Component/Notifier/Bridge/SpotHit/Tests/SpotHitTransportTest.php

```diff
@@ -42,4 +47,61 @@ public static function unsupportedMessagesProvider(): iterable
         yield [new ChatMessage('Hello!')];
         yield [new DummyMessage()];
     }
+
+    public function testShouldSendAMessageUsingTheSpotHitAPI(): void
+    {
+        $expectedRequest = [
+            function($method, $url, $options) {
+                $this->assertSame('POST', $method);
+                $this->assertSame('https://www.spot-hit.fr/api/envoyer/sms', $url);
+                $this->assertSame('key=&destinataires=0611223344&type=premium&message=Hello%21&expediteur=', $options['body']);
+
+                return new MockResponse(json_encode([
+                    'resultat' => ['success' => 'true'],
+                    'id' => '???',
+                ], JSON_THROW_ON_ERROR));
+            }
+        ];
+
+        $client = new MockHttpClient($expectedRequest);
+        $transport = new SpotHitTransport('', '', $client, null);
+        $transport->send(new SmsMessage('0611223344', 'Hello!'));
+    }
+
+    public function argumentsProvider(): Generator
+    {
+        yield [
+            function(SpotHitTransport $transport) { $transport->setSmsLong(true); },
+            function(array $bodyArguments) { $this->assertSame('1', $bodyArguments['smslong']); },
+        ];
+
+        yield [
+            function(SpotHitTransport $transport) { $transport->setLongNBr(3); },
```

Comment:
Please add a TestCase where we send both parameters

Reply:
Done, below